### PR TITLE
Feature visual aesthetics

### PR DESCRIPTION
**Pull request by flbulgarelli**
* Increased font sizes and heights to improve readability 
* Using a very different call to action salmon color. 
* Bigger submit button
* Centering CTA button

### CODE REVIEW

**Comment by flbulgarelli** [comment]
step1

step2


step3

step4

File: app/assets/stylesheets/application/_colors.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
-$brand-complementary:   #DB9634;
-$brand-primary:         #2C4250;
+$brand-complementary:   #FF5B81;
```
Comment (aguspina) [inline app/assets/stylesheets/application/_colors.scss]: I love this!!

File: app/helpers/concerns/with_breadcrumbs.rb [hunk 2]
```diff
@@ -18,6 +18,6 @@ def guide_breadcrumb(g)
   end
 
   def path_breadcrumb(p)
-    "<li>#{link_to_chapter p}</li>".html_safe
+    "<li><span class=\"ahahamojimoji\">#{link_to 'ム', root_path }</span></li><li>#{link_to_path_element p}</li>".html_safe
```
Comment (aguspina) [inline app/helpers/concerns/with_breadcrumbs.rb]: Great idea, the new breacrumb is awesome!

File: app/views/chapters/index.html.erb [hunk 3]
```diff
@@ -1,20 +1,46 @@
...
+  <div class="mumuki">
+    <h1 class="logo"><span class="ahahamojimoji">ム</span></h1>
+
+    <h1>Mumuki</h1>
+
+    <h2><%= t :mumuki_catchphrase %></h2>
+  </div>
+
+  <div class="text-box">
+    <p>¿Alguna vez te sucedió estar haciendo una tarea tediosa y repetitiva? ¿Te descrubiste a vos mismo cometiendo
```
Comment (aguspina) [inline app/views/chapters/index.html.erb]: This should be internationalized @flbulgarelli

Comment (flbulgarelli) [inline app/views/chapters/index.html.erb]: :laughing:

Comment (gastonprieto) [inline app/views/chapters/index.html.erb]: You have a typo, you wrote ```descrubiste```

Comment (flbulgarelli) [inline app/views/chapters/index.html.erb]: Yup :disappointed: 

Actually, that should be a configurable text per tenant

**Comment by flbulgarelli** [comment]
I removed the preface change. I will go in a different PR.

**Pull request merged**